React/wbooks/src/redux/rents/reducer.js

```diff
@@ -33,15 +33,28 @@ export default function reduce(state = initialState, action = {}) {
       return state.merge({
         localRents: action.payload
       });
-    case types.RENTS_WISHES_FETCHED:
+    case types.RENTS_WISHES_FETCH:
       return state.merge({
-        wishes: action.payload
+        loading: true
       });
-    case types.RENTS_WISHES_SAVING:
+    case types.RENTS_WISHES_FETCH_FAILURE:
+      return state.merge({
+        loading: false
+      });
+    case types.RENTS_WISHES_FETCH_SUCCESS:
+      return state.merge({
+        wishes: action.payload,
+        loading: false
+      });
+    case types.RENTS_WISHES_SAVE:
       return state.merge({
         isLoading: true
       });
-    case types.RENTS_WISHES_SAVED:
+    case types.RENTS_WISHES_SAVE_SUCCESS:
+      return state.merge({
+        isLoading: false
+      });
+    case types.RENTS_WISHES_SAVE_FAILURE:
```

Comment:
When are a many indenticall reducers for many actions, u can put them together...
```
case types.RENTS_WISHES_SAVE_SUCCESS:
case types.RENTS_WISHES_SAVE_FAILURE:
case types.ANOTHER_ACTION:  
    return ( ... );